backend/workflow_manager/workflow/workflow_helper.py

```diff
@@ -145,8 +146,13 @@ def process_input_files(
                     ExecutionStatus.STOPPED, error=str(exception)
                 )
                 break
-        if error_raised and error_raised == total_files:
-            execution_service.update_execution(ExecutionStatus.ERROR)
+            except Exception as error:
+                error_message = str(error)
+                error_raised += 1
+        if error_raised and error_raised >= processed_files:
```

Comment:
@muhammad-ali-e why do we check for `error_raised >= processed_files`?

Reply:
To ensure that exceptions were raised for all the files that were processed. So we can say the execution status as ERROR.